waltz-tools/src/main/java/com/wepay/waltz/tools/client/ClientCli.java

```diff
@@ -473,6 +370,124 @@ private static int nextExponentialDistributedInterval(int avgInterval) {
         }
     }
 
+    /**
+     * Use {@code Validate} command to submit transactions to Waltz
+     * server for testing, and consume the transactions for validation,
+     * including transaction data and optimistic locking.
+     *
+     * When running {@code Validate} command, there should not be any external
+     * client that is producing TXNs of the same partition (different partition
+     * is okay). Otherwise, the validation result will be unpredictable.
+     */
+    private static final class Validate extends ClientCliBaseClass {
+        private static final String NAME = "validate";
+        private static final String DESCRIPTION = "Submit transactions for validation";
+        private static final int DEFAULT_NUMBER_ACTIVE_PARTITIONS = 1;
```

Comment:
Why not have this variable as part of ClientCli base class? You seem to be using it in `Validate`, `create-producer` and `create-consumer` CLIs.